helm/cluster-cloud-director/values.yaml

```diff
@@ -44,10 +44,8 @@ network:
     vipSubnet: ""  # Virtual IP CIDR for the external network.
 
 proxy:  # (Optional) Defines HTTP proxy environment variables.
+  secretName: ""
   enabled: false
-  httpProxy: ""
-  httpsProxy: ""
-  noProxy: ""  # Bypass the proxy for specific hosts - cluster IP range for instance.
```

Comment:
If we want to deliver it like that here I would like it this secret to be documented in the repo and remove all references of `cluster-apps-operator` in the PR. This is a public repo and can be used by anyone in the wild. We should not make it confusing for potential users. For the GS platform we can document the inner working in our official docs.

Reply:
@kopiczko 

> If we want to deliver it like that here I would like it this secret to be documented in the repo

with documentation you mean a comment at the end of the line? If so, will do it that way as i didn't find any other "documentation" in this repo where the values are documented.

> This is a public repo and can be used by anyone in the wild. 

Unfourtanetly we have multiple other giantswarm helm charts where we refer to values from `cluster-apps-operator`/`cluster-operator` where it's not that easy for potential customers to re-use the helm-chart.
If this is still the goal for all our repos (what i assume) i guess we hit this topic already in the past (as other apps are using values from `cluster-apps-operator`/`cluster-operator` for months/years) i'm wondering if there was a decission how to solve this topic in general?!

Reply:
> Unfourtanetly we have multiple other giantswarm helm charts where we refer to values from `cluster-apps-operator`/`cluster-operator` where it's not that easy for potential customers to re-use the helm-chart.

Is this a reason to make yet another chart like that? We already have a fork of `cluster-openstack` [in the wild](https://github.com/eschercloudai/cluster-openstack) so it isn't exactly true. 

BTW isn't `<clusterName>-cluster-values` linked in the cluster App already? Maybe we can stick with the previous values schema?